### ISSUE TITLE
Safari 6.1.6 not triggering notification

### ISSUE THREAD

**Issue by ajshapiro** [browser-update/browser-update#357]
This user agent (selected as "Safari 6.1" in User Agent menu) triggers notification:
Mozilla/5.0 (Macintosh; Intel Mac OS X 10_8_4) AppleWebKit/537.78.2 (KHTML, like Gecko) Version/6.1 Safari/537.78.2

This user agent (selected as "Default (Automatically Chosen)" does not trigger a notification:
Mozilla/5.0 (Macintosh; Intel Mac OS X 10_7_5) AppleWebKit/537.78.2 (KHTML, like Gecko) Version/6.1.6 Safari/537.78.2

Any ideas why?

**Comment by kaptainkommie** [comment]
The only difference I can see besides the OSX version is the `Version/6.1` versus `Version/6.1.6`  in the default.

**Comment by josselex** [comment]
What code are you using? Are you saying the browsers are detected differently? Are you sure you cleared your cookies when testing?

**Comment by ajshapiro** [comment]
Yes I cleared cookies when testing each time. When testing with User Agent set to "Default (Automatically Chosen)" in Safari 6.1.6, I do not get any browser update notification at all.  When I change the User Agent setting to "Safari 6.1", I do get the browser update notification. The corresponding actual user agents for each setting are shown in my original post.

This is the code:
`    <script> 
    var $buoop = {vs:{i:10,f:30,o:37,s:8,c:40},reminder:0,api:4,text:'Your browser ({brow_name}) is outdated. You must <span class="buorgHighlight">update your browser</span> to use Suppose. &nbsp;&nbsp;<a{up_but}>Update Browser</a> &nbsp; <a{ignore_but}>Ignore</a>'}; 
    function $buo_f(){ 
    var e = document.createElement("script"); 
    e.src = "//browser-update.org/update.min.js"; 
    document.body.appendChild(e);
    };
    try {document.addEventListener("DOMContentLoaded", $buo_f,false)}
    catch(e){window.attachEvent("onload", $buo_f)}
    </script>`

**Comment by josselex** [comment]
Hi @ajshapiro

I figured out the reason.
The second user agent is on mac Mac OS 10.7. 
For this versions, there is NO up to date browser available and the system can not be updated on this old macs, so we do not notify the user because he cannot update anyway.

What do you think of this?

**Comment by ajshapiro** [comment]
OK, I see overriding the User Agent in Safari does also use a later Mac OS version as well.  I do think it would be good to show a message even for an older OS like Mac OS 10.7 so the user knows the site may not be fully functional.  Also, this particular Mac with OS 10.7 CAN be upgraded to a newer OS, so in this case, there is an upgrade path that would allow the user to use as a supported browser, so might as well notify them. Or they might also be able to use another computer that does have a supported browser.

So bottom line, I still think it would be best to show the notification.
